### PR TITLE
Update the schema document for the hera_auto_spectrum table

### PR DESCRIPTION
**Pull request by bhazelton**
## Description

Updating the schema document was missed in #640.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Schema change (any change to the SQL tables)
- [ ] New feature without schema change (non-breaking change which adds functionality)
- [ ] Change associated with a change in redis structure
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Version change
- [ ] Build or continuous integration change
- [x] Other

## Checklist:

Other:
- [x] My code follows the code style of this project.
- [x] I understand the updates required onsite (detailed in the readme) and I will make those
changes when this is merged.
- [ ] Unit tests pass **on site** (This is a critical check, CI can differ from site).
- [ ] I have updated the [CHANGELOG](https://github.com/HERA-Team/hera_mc/blob/main/CHANGELOG.md).

### CODE REVIEW

**Comment by bhazelton** [comment]
Both the update tex and pdf file are on this PR.

**Comment by mkolopanis** [comment]
oh i see it! the line was so small when it was collapsed!

File: docs/mc_definition.tex [hunk 1]
```diff
@@ -770,10 +770,22 @@ \subsubsection{hera\_autos}
 \textbf{antenna\_number} & int & antenna number \\ \hline
 \textbf{antenna\_feed\_pol} & string & antenna feed polarization, either `e' or `n'. \\ \hline
 measurement\_type* & string & Currently can only be `median'. \\ \hline
-value* & float & Measured value. \\ \hline
+value* & float & calculated value (median across frequencies). \\ \hline
 \end{tabular}
 \end{center}
 
+\subsubsection{hera\_auto\_spectrum}
```
Comment (mkolopanis) [inline docs/mc_definition.tex:777]: I see this uses the same subsubsection heading as the hera_autos, but I don't see a `hera_auto_spectrum` line in the table of contents for section 4 in the PDF. Can you verify it's absence or presence on your end?

File: docs/mc_definition.tex [hunk 2]
```diff
@@ -770,10 +770,22 @@ \subsubsection{hera\_autos}
...
+Antenna autocorrelation spectra.
+\begin{center}
+\begin{tabular}{| p{4cm} | p{2cm} | p{10cm} |}
+\hline
+ {\bf Column} & {\bf Type}  & {\bf Description} \\ [0.5ex]  \hline\hline
+\textbf{time} & long & status time in floor(gps seconds)\\ \hline
+\textbf{antenna\_number} & int & antenna number \\ \hline
+\textbf{antenna\_feed\_pol} & string & antenna feed polarization, either `e' or `n'. \\ \hline
+spectrum* & array of float & autocorrelation spectrum. \\ \hline
```
Comment (mkolopanis) [inline docs/mc_definition.tex:786]: do we want to use the terminology of the PSQL types and call this out as a array of reals?

**Review by mkolopanis — COMMENTED** [review]
Two small question/comments.. questaments!

File: docs/mc_definition.tex (hunk 2 above)
Comment (bhazelton) [reply docs/mc_definition.tex:786]: Sure, but I think users will in practice experience it as a float. I'll try to make that clear.

File: docs/mc_definition.tex (hunk 1 above)
Comment (bhazelton) [reply docs/mc_definition.tex:777]: good catch!! I needed to compile one more time to make all the references work their way through.

**Review by mkolopanis — APPROVED** [review]
Awesome thanks for the updates! looks good to me but now I'm scared of merging things too early :laughing:

**Pull request merged**